### PR TITLE
EASYOPAC-1146 - Fix inconsistency of availability types.

### PR DESCRIPTION
**Pull request by bozeak**
#### Link to issue

https://inlead.atlassian.net/browse/EASYOPAC-1146

#### Description

On search result collection items was observed inconsistency in rendering of materials availability labels.

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

### CODE REVIEW

File: modules/ding_availability/js/ding_availability_labels.js [hunk 1]
```diff
@@ -191,15 +188,6 @@
           if (status[i] === true) {
             class_name = i;
           }
-          else {
```
Comment (guddo) [inline modules/ding_availability/js/ding_availability_labels.js]: Why are this removed?

Comment (bozeak) [reply modules/ding_availability/js/ding_availability_labels.js]: This was removed because on each ajax request done during one page load this function was triggered and to each non-online item in collection was added all classes (available, unavailable, reservable, not-reservable), fact which also was breaking ding_availability functionality.

Comment (guddo) [reply modules/ding_availability/js/ding_availability_labels.js]: Ok, but how do items get un-available and not-reserveable now?

Comment (bozeak) [reply modules/ding_availability/js/ding_availability_labels.js]: The classes with `true` value only will be added.

Comment (guddo) [reply modules/ding_availability/js/ding_availability_labels.js]: I guess that there was a purpose by adding a class called `not-available` to items that is not available, as well as `un-reserveable` to items that can not be reserved.

Now again; what is the purpose of removing this? Does it improve performance, or?

Comment (bozeak) [reply modules/ding_availability/js/ding_availability_labels.js]: I've removed those because:
1. I didn't found any logic there those classes was used, so we got rid of dead code.
2. This condition wasn't working as it was expected. Classes, as I wrote in 1-st reply, was added to each element despite the fact that those classes should not have been added there. Skipping this part, I can treat as an improvement.
